howtos/vs2017_cmake.rst

```diff
@@ -154,16 +154,13 @@ Using tasks with tasks.vs.json
 Another alternative is using file `tasks <https://docs.microsoft.com/en-us/visualstudio/ide/customize-build-and-debug-tasks-in-visual-studio?view=vs-2017>`_
 feature of Visual Studio 2017. This way you can install dependencies by running :command:`conan install` as task directly in the IDE.
 
-All you need is to right click on your *conanfile.py*-> Configure Tasks (see the
+All you need is to right click on your *conanfile.py* -> Configure Tasks (see the
 `link above <https://docs.microsoft.com/en-us/visualstudio/ide/customize-build-and-debug-tasks-in-visual-studio?view=vs-2017>`_) and add the
 following to your *tasks.vs.json*.
 
 .. warning::
 
-    The file *tasks.vs.json* is added to your local *.vs* folder so it is not supposed to be added to your version control system. There is
-    also a feature
-    `request <https://visualstudio.uservoice.com/forums/121579-visual-studio-ide/suggestions/33814138-add-macro-buildroot-to-tasks-vs-json>`_
```

Comment:
I couldn't find the new link for this one so I removed as I didn't seem so crucial